### PR TITLE
[MRG] FutureWarning: view_stats_map() is under active development; may change without warning

### PR DESCRIPTION
**Pull request by kchawla-pi**
For #1778 
Opening a new PR for this, the other one was too outdated and problematic.

### CODE REVIEW

File: nilearn/plotting/html_stat_map.py [hunk 1]
```diff
@@ -75,6 +76,15 @@ def _rgb_to_papaya(x, cmap):
     return cmap
 
 
+def _view_stats_map_change_warning():
+    change_warn_msg = (
+        '"{}" is under actve development and its interface and functionality '
+        'may change in future without warning.\n Please do not rely on it'
+        'in critical and production environments.'
+    ).format('view_stat_map')
```
Comment (GaelVaroquaux) [inline nilearn/plotting/html_stat_map.py]: The format is not useful. You can just modify the string.

**Comment by GaelVaroquaux** [comment]
The failure is telling you that we have another type of warning that is creeping in.

**Comment by kchawla-pi** [comment]
Yeah looking into that.
Will incorporate the code reviews after I have fixed this.
Another issue is that there are 7 calls for `view_stat_map()` but only 6 warnings are generated.
One of them does not generate the warning in this context, but does so when called standalone.

**Comment by kchawla-pi** [comment]
This test generates two types of warnings: 
1. `UserWarning` by `image.resample_img()`
2. `FutureWarning` by `view_stat_map()`

In Python3.4 the first warning is being generated multiple times, in other builds it is being generated only the first time (which my context manager deliberately doesn't catch).

**Comment by kchawla-pi** [comment]
In the next commit, I intend to check that only the specific User & Future warning is asserted, but not their order or count.
In effect, `assert_true(set(warning_.category for warning_ in w) == set(FutureWarning, UserWarning)`
but I will also check for the message, not just categories.

**Comment by kchawla-pi** [comment]
Not checking the message, only the category.

**Comment by kchawla-pi** [comment]
@GaelVaroquaux THis okay?

**Comment by GaelVaroquaux** [comment]
LGTM. Merging. Thank you

**Pull request merged**